internal/cmd/io_binary.go

```diff
@@ -105,6 +115,34 @@ func NewBinaryIO(ctx context.Context, id string, uri *url.URL) (_ UpstreamIO, er
 	}, nil
 }
 
+// sanitizePath parses the URL object and returns an absolute path to the logging driver
+func sanitizePath(uri *url.URL) (sanitized string) {
+	path := uri.Path
+	// Convert windows-style slashes to linux-style to clean the path
+	path = strings.Replace(path, "\\", "/", -1)
+	path = filepath.Clean(path)
+	// Path is clean, convert back
+	sanitized = strings.Replace(path, "/", "\\", -1)
```

Comment:
nit: You can just keep reassigning `path` here, instead of introducing `sanitized`.